modules/ROOT/pages/organization.adoc

```diff
@@ -176,4 +183,9 @@ Users who are assigned the Organization Administrators role in the master organi
 
 === Deleting Business Groups
 
-Only an organization administrator can delete business group. The master (root) organization cannot be deleted, even by an organization administrator.
+Only an organization administrator can delete business groups. The master (root) organization cannot be deleted, even by an organization administrator.
```

Comment:
Here's a place where contraction reads more naturally/easily than the spelled out version. Just me?